### PR TITLE
Activate streaming from object storage

### PR DESCRIPTION
**Pull request by mdellweg**
fixes #493

### CODE REVIEW

File: pulp_ansible/app/tasks/collections.py [hunk 1]
```diff
@@ -301,18 +298,24 @@ def import_collection(
...
                     get_file_obj_from_tarball(tar, "FILES.json", temp_file.file.name)
                 )
-            url = _get_backend_storage_url(artifact_file)
-            importer_result = process_collection(
-                artifact_file, filename=filename, file_url=url, logger=user_facing_logger
-            )
+            with open(local_temp_file.name) as file_handle:
+                importer_result = process_collection(
+                    file_handle, filename=filename, logger=user_facing_logger
+                )
```
Comment (mdellweg) [inline pulp_ansible/app/tasks/collections.py:318]: This is a hell of a workaround to prevent galaxy-importer to try to be clever.

Comment (fao89) [reply pulp_ansible/app/tasks/collections.py:318]: FYI @awcrosby

Comment (awcrosby) [reply pulp_ansible/app/tasks/collections.py]: haha. we may have had a reason for this originally, but if we are able to get it working with pulp_ansible passing a file handle to galaxy-importer, then +1 to that

**Comment by mdellweg** [comment]
I intend to solve the issue on the galaxy-importer side.

File: pulp_ansible/app/tasks/collections.py [hunk 2]
```diff
@@ -397,7 +393,10 @@ def create_collection_from_importer(importer_result, metadata_only=False):
 
 def _get_backend_storage_url(artifact_file):
     """Get artifact url from pulp backend storage."""
-    if settings.DEFAULT_FILE_STORAGE == "pulpcore.app.models.storage.FileSystem":
+    if (
+        settings.DEFAULT_FILE_STORAGE == "pulpcore.app.models.storage.FileSystem"
+        or not settings.REDIRECT_TO_OBJECT_STORAGE
+    ):
```
Comment (fao89) [inline pulp_ansible/app/tasks/collections.py:399]: 👍🏽

**Review by fao89 — COMMENTED** [review]
shouldn't you ensure galaxy-importer version [here](https://github.com/pulp/pulp_ansible/blob/main/requirements.txt#L3)?

**Review by fao89 — APPROVED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 aiofiles>=0.8.0
 async_lru~=1.0
-galaxy_importer>=0.4.1
+galaxy_importer>=0.4.5
```
Comment (fao89) [inline requirements.txt:3]: their release failed: https://github.com/ansible/galaxy-importer/runs/6479389918?check_suite_focus=true

Comment (awcrosby) [reply requirements.txt:3]: galaxy-importer 0.4.5 is now on pypi

**Review by awcrosby — APPROVED** [review]

**Pull request merged**